src/zql/ast-to-ivm/pipeline-builder.test.ts

```diff
@@ -8,7 +8,7 @@ import {buildPipeline} from './pipeline-builder.js';
 const e1 = z.object({
   id: z.string(),
   a: z.number(),
-  b: z.bigint(),
+  b: z.number(),
```

Comment:
Few things:

- I think there is bigint support elsewhere in the codebase. I agree we probably don't want to support it, but shouldn't we remove impl at same time?
- I think the intent here is to cover all the supported types. `number` is already covered by `a`. Should we just reduce the number of fields?